### PR TITLE
fix bug for istio locality illegal characters

### PR DESCRIPTION
**Pull request by tanjunchen**
fixed https://github.com/slime-io/slime/issues/404

### CODE REVIEW

File: bin/publish.sh [hunk 1]
```diff
@@ -102,7 +102,7 @@ for action in $actions; do
   case "$action" in
   build)
     echo "go build submodules ${MOD}"
-    CGO_ENABLED="${CGO_ENABLED}" GOOS="${TARGET_GOOS}" GOARCH=${TARGET_GOARCH} go build -o manager.exe
+    CGO_ENABLED="${CGO_ENABLED}" GOOS="${TARGET_GOOS}" GOARCH=${TARGET_GOARCH} go build -o manager
```
Comment (MouceL) [inline bin/publish.sh:105]: I don't think this needs to be modified

With .ignore, avoid binary uploads.

**Comment by believening** [comment]
Not sure if there are other places where labels have been added by slime that haven't been validated. The current fix is effective for the problems encountered by #404.

File: bin/publish.sh (hunk 1 above)
Comment (tanjunchen) [reply bin/publish.sh:105]: OK，I will remove it.

**Comment by tanjunchen** [comment]
> Not sure if there are other places where labels have been added by slime that haven't been validated. The current fix is effective for the problems encountered by #404.

updated. @believening @MouceL

**Review by believening — APPROVED** [review]

**Pull request merged**